### PR TITLE
Components

### PR DESCRIPTION
**Pull request by MasumaJaffery**
In this Milestone;
1. Add Components.
2. Use of Modules.
3. Add Styling.
Thanks for reviewing my PR!
Happy Coding 🤓

### CODE REVIEW

File: src/App.js
Comment (joskalenda) [inline src/App.js:1]: I think you did wrong in deleting this file, I think it is required that you import your calculator component into the App.js file but sorry to see that in the source code, you are not doing so, please refer to the requirement screenshot bellow

File: src/components/style.css
Comment (joskalenda) [inline src/components/style.css:1]: - [ ] Your app is running, however, make sure you use the correct style properties as per the original design please, you are using the green color which in the design is orange.
- [ ] make sure you align the 0 at the end of the input, not the begining

YOUR DESIGN




FIGMA

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @MasumaJaffery , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @MasumaJaffery, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the changes requested by the previous review👏
- calculator matches the wireframe👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**